### PR TITLE
WINDUP-766: Scan JSP's for potential migration issues

### PR DESCRIPTION
**Pull request by jsight**
https://issues.jboss.org/browse/WINDUP-766

### CODE REVIEW

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/435/
Test PASSed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/1065/
Test PASSed.

**Comment by Maarc** [comment]
I tested the PR Yesterday and it seemed to work well! I have not reviewed the code.

File: rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/AnalyzeJSPFileRuleProvider.java [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            {
+                // Setup some basic details about the "Java Class"
+                // source root, is decompiled, javaclass. package name
+                sourceFile.setPackageName("");
+                sourceFile.setDecompiled(false);
+                JavaClassService javaClassService = new JavaClassService(event.getGraphContext());
+                JavaClassModel classModel = javaClassService.create();
+                classModel.setPackageName("");
+                classModel.setSimpleName(sourceFile.getFileName());
+                classModel.setQualifiedName(sourceFile.getFileName());
```
Comment (mbriskar) [inline rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/AnalyzeJSPFileRuleProvider.java]: simple/qualified name are the same?

**Comment by mbriskar** [comment]
There is a need to change .xsd and ideally maybe one test in windup.xml format also.

**Comment by jsight** [comment]
Why would the xsd need to change?

File: rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/AnalyzeJSPFileRuleProvider.java (hunk 1 above)
Comment (jsight) [inline rules-java-ee/addon/src/main/java/org/jboss/windup/rules/apps/javaee/rules/AnalyzeJSPFileRuleProvider.java]: Yes, as the package name is marked as empty. It's a bit of a hack as it is hard to know what the generated package name will really be.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-core-PR-dependents-check/446/
Test FAILed.

**Comment by windupgithubbot1** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://10.3.8.5//job/windup-pr-builder/1078/
Test FAILed.

**Pull request merged**